### ISSUE TITLE
Several channels are not loading (e. g. UK, USA, Poland, Austria and some others (Hypnotix 4.3)

### ISSUE THREAD

**Issue by paintdog** [linuxmint/hypnotix#317]
I'm using Hynotix 4.3 under Linux Mint 21.3 Cinnamon.

After an update to the new version some days ago I've now much more channels but the UK channels are not loaded for me (I'm in Germany). The German channels load perfectly and I enjoy sound and picture. The same problem occurs with the channels from the USA, Poland, the Netherlands, Austria and probably some others (Monaco works). For Servus TV from Austria I got the error message ("Free-TV is not responding.").

Any idea what causes the problem? Any update planned?

**Comment by paintdog** [comment]
I started hypnotix in my console:
``
user@linux:~$ hypnotix
System version of yt-dlp selected.
user@linux:~$ Free-TV: 1584 channels, 88 groups, 0 series, 232 movies
Loading providers...
Free-TV: Getting playlist...
Free-TV: Checking playlist...
Free-TV: Loading channels...
No flag found for: Documentaries (AR)
No flag found for: Documentaries (EN)
No flag found for: News (AR)
No flag found for: News
No flag found for: Business
No flag found for: News (ES)
CHANNEL: 'BBC One' (http://92.114.85.81:8000/play/a001/index.m3u8)
Loading providers...
Free-TV: Downloading playlist...
4194304 bytes
Free-TV: Checking playlist...
Free-TV: 1584 channels, 88 groups, 0 series, 232 movies
Free-TV: Loading channels...
CHANNEL: 'Das Erste Ⓖ' (https://mcdn.daserste.de/daserste/de/master.m3u8)
CHANNEL: 'Das Erste Ⓖ' (https://mcdn.daserste.de/daserste/de/master.m3u8)
Cannot load libcuda.so.1
``
It tries to connect to BBC One since several minutes (one hour?) and nothing happens. I switched to a channel in Germany and this works fine.
